src/lib.rs

```diff
@@ -1503,3 +1516,7 @@ impl<T: Counter, F: Counter> PartialEq<Histogram<F>> for Histogram<T>
 #[path = "tests/tests.rs"]
 #[cfg(test)]
 mod tests;
+
+/// Serialization/deserialization support.
+#[path = "serialization/serialization.rs"]
```

Comment:
My editor always shows me the full path to the open file, and I use full-path fuzzy searching to open files, so it doesn't matter whether the distinguishing part of the name is the directory or the filename to me ;)